presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -87,16 +87,16 @@ Constructors
 
```

Comment:
Regarding the commit title/description, it's not really "standardize", but fix the formatting. Single backquotes in rst are used for "interpreted text" (e.g., what we use for `:func:` or `:issue:`)